### ISSUE TITLE
WCAG 2.2 A: List element `.ck-aria-live-region-list` is empty

### ISSUE THREAD

**Issue by dtdesign** [ckeditor/ckeditor5#18561]
## 📝 Provide detailed reproduction steps (if any)

1. Load the official demo at https://ckeditor.com/ckeditor-5/demo/feature-rich/
2. Inspect the page and look for the `.ck-body-wrapper` beneath the `<body>` element.
3. Inside of the `.ck ck-aria-live-announcer` container are two `aria-live` sections, each containing an `ul` without any items.

### ✔️ Expected result

The list elements should either be dynamically inserted when there is something to announce or be marked as `aria-hidden` while they do not contain any content.

### ❌ Actual result

The lists exists even if there are zero list items.

This is effectively in violation of [SC 1.3.1](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html) of WCAG 2.2 level A which does not permit empty list elements, since at least one element with the (implicit) `listitem` role is required.

## 📃 Other details

* Browser: Chrome 136
* OS: macOS 15.5
* First affected CKEditor version: Unknown
* Installed CKEditor plugins: (Full online demo)

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
